apps/utils/locust_tests/config/regression_suites_compare_meta.json

```diff
@@ -0,0 +1,162 @@
+{
+  "totals": [
+    {
+      "name": "num_requests",
+      "better_if": "LARGER",
+      "warn_percent": 150,
+      "fail_percent": 200
+    },
+    {
+      "name": "median_response_time",
+      "better_if": "SMALLER",
+      "warn_percent": 150,
+      "fail_percent": 200
+    },
+    {
+      "name": "average_response_time",
+      "better_if": "SMALLER",
+      "warn_percent": 150,
+      "fail_percent": 200
+    },
+    {
+      "name": "min_response_time",
+      "better_if": "SMALLER",
+      "warn_percent": 2500,
+      "fail_percent": 5000
+    },
+    {
+      "name": "max_response_time",
+      "better_if": "SMALLER",
+      "warn_percent": 150,
+      "fail_percent": 500
+    },
+    {
+      "name": "total_reqs_per_second",
+      "better_if": "LARGER",
+      "warn_percent": 150,
+      "fail_percent": 200
+    },
```

Comment:
With the way that locust works, this is essentially a side effect of the response time, isn't it? Seems like we'd be capturing two metrics that basically say the same thing with requests/second.

Reply:
I suppose it is, although I don't think it hurts to include this particular metric in our baseline comparisons and in the aggregated stats we store. It's been a good "at-a-glance" metric for when we run look at logs to see if a run is overall good or not, too.